src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+/* eslint-disable react/jsx-one-expression-per-line */
+// "react/jsx-one-expression-per-line": [<enabled>, { "allow": "none"|"literal"|"single-child" }];
+// eslint-disable-next-line react/jsx-one-expression-per-line
+
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => {
+  const gender = sex;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is {name}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          I am {age}
+        </p>
+      )}
+
+      {sex && isMarried ? (
```

Comment:
The 'sex' check in the condition is not necessary since 'sex' is always expected to be a string (either 'm' or 'f'). Only checking 'isMarried' should suffice.